src/OrchardCore.Modules/OrchardCore.Lists/OrchardCore.Lists.csproj

```diff
@@ -13,6 +13,7 @@
     <ProjectReference Include="..\..\OrchardCore\OrchardCore.Media.Abstractions\OrchardCore.Media.Abstractions.csproj" />
     <ProjectReference Include="..\..\OrchardCore\OrchardCore.ResourceManagement.Abstractions\OrchardCore.ResourceManagement.Abstractions.csproj" />
     <ProjectReference Include="..\..\OrchardCore\OrchardCore.MetaWeblog.Abstractions\OrchardCore.MetaWeblog.Abstractions.csproj" />
+    <ProjectReference Include="..\OrchardCore.AdminTrees\OrchardCore.AdminTrees.csproj" />
```

Comment:
Modules don't reference modules. Create an abstraction project if you need to expose something. And add a feature dependency.

Reply:
Done. But not sure if I should add admintree as a dependency in the manifest file. I just added a project reference pointing to the abstraction. I did it for the List and Contents modules., of course.